cockatrice/src/player.cpp

```diff
@@ -3141,7 +3147,8 @@ void Player::actSetAnnotation()
     bool ok;
     dialogSemaphore = true;
     QString annotation = QInputDialog::getMultiLineText(game, tr("Set annotation"),
-                                                        tr("Please enter the new annotation:"), oldAnnotation, &ok);
+                                                        tr("Please enter the new annotation:"), oldAnnotation, &ok)
+                             .left(MAX_NAME_LENGTH);
```

Comment:
Is this what clangify does? Weird

Reply:
yes, there is no proper way to validate this type of widget so I just slapped left on it